src/Scrapers/CFScraper.py

```diff
@@ -12,7 +12,12 @@
 class CFScraper(Scraper):
     def __init__(self, url) -> None:
         super().__init__(url)
-        response = requests.get(url)
+        # Because request only makes http request calls, change our https link to http
+        protocol , link = url.split(':')
+        protocol = 'http'
+        url = protocol + ':'+link
+        proxy = Scraper(url).get_proxy()
```

Comment:
Kindly read documentation regarding inheritance in Python. Here you are simply creating a new object of Scraper instead of using the inherited methods.